api/cache.go

```diff
@@ -20,6 +20,8 @@ const (
 	PipelineCache CacheNamespace = "pl"
 	// IssueCache is the cache type of issues.
 	IssueCache CacheNamespace = "is"
+	// LicenseCache is the cache type of license.
+	LicenseCache CacheNamespace = "l"
```

Comment:
I guess we don't need this, it's only used if we store license in the sqlite db (in the store module)